### PR TITLE
Migrate VsTestV2 task to Node10

### PR DESCRIPTION
**Pull request by mr-dokara**
**Task name**: VsTestV2

**Description**: Migrated VsTestV2 to Node10
- Execution handler is updated
- Typings are deleted

**Documentation changes required:** N

**Added unit tests:** N

**Attached related issue:** N

**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [x] Checked that applied changes work as expected

### CODE REVIEW

File: Tasks/VsTestV2/task.loc.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
   "author": "Microsoft Corporation",
   "version": {
     "Major": 2,
-    "Minor": 198,
+    "Minor": 199,
```
Comment (anatolybolshakov) [inline Tasks/VsTestV2/task.loc.json:20]: Please bump to 200

**Review by anatolybolshakov — APPROVED** [review]
LGTM, just leaved one small comment also

**Review by EzzhevNikita — APPROVED** [review]

**Comment by golubev-ivan** [comment]
cc @geekzter for visibility

**Comment by anatolybolshakov** [comment]
@golubev-ivan should legacy tests be moved to L0 tests of the task also?
image

**Review by rasunkar — APPROVED** [review]

**Pull request closed without merging**